features/compose_message.feature

```diff
@@ -0,0 +1,30 @@
+Feature: As a visitor.
+  So that I can use the service.
+  I would like to be able to go to login page and login on my account.
```

Comment:
Your headline for this PR would indicate that you are *not* testing login but are instead testing composing and sending a message. One is wrong, please fix.